bubblesub/cmd/seek.py

```diff
@@ -27,7 +28,10 @@ class SeekCommand(BaseCommand):
 
     @property
     def is_enabled(self) -> bool:
-        return self.api.playback.is_ready
+        return self.api.playback.is_ready and (
+            self.api.gui.current_view == ViewState.Video
+            or self.api.gui.current_view == ViewState.Full
```

Comment:
I should be able to seek even if the  video is hidden so that I can screenshot (from commands) at the time I want